src/openassetio/managerAPI/ManagerInterface.cpp

```diff
@@ -0,0 +1,4 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2013-2022 The Foundry Visionmongers Ltd
+
+#include <openassetio/managerAPI/ManagerInterface.hpp>
```

Comment:
No blank line at EOF?

Reply:
Added blank lines and updated `.editorconfig` in f81c11d